TestApp/app/screens/CrashesScreen.js

```diff
@@ -0,0 +1,214 @@
+import React, { Component } from 'react';
+import { Image, View, Text, TextInput, Switch, SectionList, TouchableOpacity } from 'react-native';
+import { DialogComponent } from 'react-native-dialog-component';
+import ImagePicker from 'react-native-image-picker';
+
+import Crashes from 'appcenter-crashes';
+
+import { FooClass } from '../../js/FooClass';
+import AttachmentsProvider from '../AttachmentsProvider';
+import SharedStyles from '../SharedStyles';
+
+export default class CrashesScreen extends Component {
+  static navigationOptions = {
+    // eslint-disable-line global-require
```

Comment:
Remove this comment and do the same as https://github.com/Microsoft/AppCenter-SDK-React-Native/pull/376/files#r211334399